ush/python/ocean/storm_tempZ40m.py

```diff
@@ -121,8 +120,8 @@
    plt.suptitle(storm.upper()+tcid.upper()+'  '+'Ver Hr '+"%03d"%(fhr)+'  (IC='+cycle+'): '+var_name+ ' & Change '+units,fontsize=15)
 
    ax121 = plt.subplot(121)
-   #(var1[k]*dumb).plot.contourf(levels=np.arange(0,850,50),cmap='RdBu_r')
-   kw = dict(levels=np.arange(np.floor(np.nanmin(var)),np.round(np.nanmax(var),1)+delta_var,delta_var))
+   kw = dict(levels=np.arange(10,25,2))
```

Comment:
The min/max values should be larger for ocean temperatures at 40 m. In the previous scripts from HSK the range was defined as 20C to 33C with a 1C contour interval.